spring-cloud-aws-messaging/src/main/java/io/awspring/cloud/messaging/core/TopicMessageChannel.java

```diff
@@ -47,6 +47,16 @@
 	 */
 	public static final String NOTIFICATION_SUBJECT_HEADER = "NOTIFICATION_SUBJECT_HEADER";
 
+	/**
+	 * Message group id for SNS message (applies only to FIFO queue).
+	 */
+	public static final String MESSAGE_GROUP_ID_HEADER = "MessageGroupId";
```

Comment:
Shouldn't this value be the same as the SQS headers in [SqsMessageHeaders.java](https://github.com/awspring/spring-cloud-aws/blob/8f2c39b371f0b34bad927340e24e89e5c5e9dfc4/spring-cloud-aws-messaging/src/main/java/io/awspring/cloud/messaging/core/SqsMessageHeaders.java)?

```suggestion
	public static final String MESSAGE_GROUP_ID_HEADER = "message-group-id";
```

Reply:
You are correct that this does not follow the naming of SqsMessageHeaders. I followed the AWS header naming. I do not see added value in adding another translation layer which is confusing to users? 
I also understand the need for consistency so maintainer can say which is preferred :)

Reply:
I would be in for keeping consistency with AWS naming, but since SQS has been implemented already like that 4 years ago, lets keep it in sync with SQS implementation. 

We can reconsider for 3.0